test/tftp/tftp_api_test.rb

```diff
@@ -17,6 +17,17 @@ def setup
     @args = { :pxeconfig => "foo" }
   end
 
+  def test_instantiate_syslinux
+    obj = app.helpers.instantiate "syslinux", "AA:BB:CC:DD:EE:FF"
```

Comment:
I'm ok with postponing the expansion of test coverage if it's coming in https://github.com/theforeman/smart-proxy/pull/414.